README.md

```diff
@@ -3,25 +3,32 @@
 The goal of this project is to add new methods to the existing Enumerable module, a method that is identical to the next ones, but (obviously) does not use the original ones.
 
 Create #my_each
-Create #my_each_with_index 
-Create #my_select 
-Create #my_all? 
+Create #my_each_with_index
+Create #my_select
+Create #my_all?
 Create #my_any?
 Create #my_none?
 Create #my_count
 Create #my_map
 Create #my_inject
```

Comment:
Replace all this with a listing of the imitated methods, like follows.

- `#each` -> `#my_each`
- `#inject` -> `#my_inject`
...